File: server/Checkout Express/src/templates/CxCardBack.java
package templates;

public class CxCardBack {
	public static String run(Object section, int num) {
		return	"<div class=\"'card back\"'>"+
			"\t<h2>"+(section)+"</h2>"+
			"\t<h3>#"+(num)+"</h3>"+
			"\t<h4>Turn card over to pay with your phone</h4>"+
			"</div>";
	}
}

File: misc/js/templates/cxCardBack.js
var template = template || {};

template.cxCardBack = function(section, num) {
	return	"<div class=\"card back\">\r\n"+
			"\t<h2>"+(section)+"</h2>\r\n"+
			"\t<h3>#"+(num)+"</h3>\r\n"+
			"\t<h4>Turn card over to pay with your phone</h4>\r\n"+
			"</div>";
};